front/src/applications/stdcm/hooks/useStdcm.ts

```diff
@@ -87,74 +93,260 @@ const useStdcm = ({
     }
   };
 
+  const getPayloadWithTolerance = (payload: PostTimetableByIdStdcmApiArg, tolerance: number) => ({
+    ...payload,
+    body: {
+      ...payload.body,
+      steps: payload.body.steps.map((step) => modifyStepTolerance(step, tolerance)),
+    },
+  });
+
+  const createSimulation = async (
```

Comment:
This function could be more complete:
- you can create the stdcmTrain with the response directly in the function (if the request is successful)
- you don't need the index and the creationDate as you set them in the action addStdcmSimulation
- the options `{ alternativePath = false, upstreamPath = false }` could be replaced by a simple string `alternativePath?: 'upstream' | 'downstream' }`